remote-method/index.js

```diff
@@ -0,0 +1,285 @@
+      this.description = answers.description;
+      done();
+    }.bind(this));
+  },
+
+  delimEndpoints: function() {
+    this.log('Let\'s add some Endpoints now.\n');
+  },
+
+  askForEndpoints: function() {
+    var done = this.async();
+
+    var prompts = [
+      {
+        name: 'httpPath',
+        message: 'Enter path for endpoints',
+        required:true
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.httpPath == null || answers.httpPath === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'httpVerb',
+          message: 'HTTP verb:',
+          type: 'list',
+          choices: ['post', 'put', 'delete', 'get', {
+            name: '(other)',
+            value: null
+          }]
+        },
+        {
+          name: 'customHttpVerb',
+          message:
+          'Enter the custom http verb',
+          validate: validateName,
+          when: function(answers) {
+            return answers.httpVerb === null;
+          }
+        }
+      ];
+
+      var httpPath = answers.httpPath;
+
+      this.prompt(subprompts, function(answers) {
+        https.push({
+          'httpPath': httpPath,
+          'httpVerb': answers.httpVerb || answers.customHttpVerb
+        });
+        console.log(https);
+        this.log('\nLet\'s add another endpoint.');
+        this.askForEndpoints();
+      }.bind(this));
+    }.bind(this));
+  },
+
+  delimAccepts: function() {
+    this.log('Let\'s add some Accept Arguments now.\n');
+  },
+
+  askForAccepts: function() {
+    var done = this.async();
+
+    var prompts = [
+      {
+        name: 'acceptsArg',
+        message: 'Enter name for accept argument:',
+        required:true
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.acceptsArg == null || answers.acceptsArg === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'acceptsType',
+          message: 'The type of accept argument:',
+          type: 'list',
+          choices: typeChoices
+        },
+        {
+          name: 'acceptsRequired',
+          message: 'Required?',
+          type: 'confirm',
+          default: false
+        },
+        {
+          name: 'acceptsDes',
+          message: 'Description of accept argument:',
+          required:true
+        }
+      ];
+
+      var argName = answers.acceptsArg;
+
+      this.prompt(subprompts, function(answers) {
+        accepts.push({
+          'arg': argName,
+          'type': answers.acceptsType,
+          'required': answers.acceptsRequired,
+          'description': answers.acceptsDes
+        });
+        this.log('\nLet\'s add another accept argument.');
+        this.askForAccepts();
+      }.bind(this));
+    }.bind(this));
+  },
+
+  delimReturns: function() {
+    this.log('Let\'s add some Return Arguments now.\n');
+  },
+
+  askForReturns: function() {
+    var done = this.async();
```

Comment:
Add info for the user how to break the loop. See e.g. the model generator ([source](https://github.com/strongloop/generator-loopback/blob/99c66a66d4d1e1179749d792ac3c298fd825636c/model/index.js#L170)).

```js
var done = this.async();
this.log('Enter an empty name when done.');
```